server/src/resources/vertiport/grpc.rs

```diff
@@ -5,75 +5,300 @@ mod grpc_server {
     tonic::include_proto!("grpc.vertiport");
 }
 
-pub use grpc_server::vertiport_rpc_server::{VertiportRpc, VertiportRpcServer};
-pub use grpc_server::{Vertiport, VertiportData, Vertiports};
-
-use crate::common::{Id, SearchFilter};
+use super::VertiportPsql;
+use crate::get_psql_pool;
+use crate::grpc::*;
+use crate::grpc_error;
+use crate::memdb::MEMDB_LOG_TARGET;
 use crate::memdb::VERTIPORTS;
-use tonic::{Request, Response, Status};
+use crate::memdb_info;
 use uuid::Uuid;
 
+pub use grpc_server::vertiport_rpc_server::{VertiportRpc, VertiportRpcServer};
+pub use grpc_server::{UpdateVertiport, Vertiport, VertiportData, Vertiports};
+
+use postgres_types::ToSql;
+use std::collections::HashMap;
+use tonic::{Code, Request, Response, Status};
+
 ///Implementation of gRPC endpoints
 #[derive(Debug, Default, Copy, Clone)]
 pub struct VertiportImpl {}
 
 #[tonic::async_trait]
 impl VertiportRpc for VertiportImpl {
-    ///vertiport_by_id // TODO implement. Currently returns arbitrary value
+    /// Takes an UUID string (vertiport_id) to get the matching database record.
+    ///
+    /// # Arguments
```

Comment:
Learned recently that having Arguments and Returns are sometimes discouraged in Rust function comments

1) These often accidentally fall out of sync with the implementation
2) Return type should be sufficiently described by the initial description line
3) Argument names should be self explanatory
4) Having `Result` and `Option` types make English explanations overly complicated/verbose

The Rust community has suggested that a `///` comment after a function argument should result in the generated rustdocs description of the argument, like:
```
fn start_game(
   name: &str /// The name of the game
) { ... }
```